### ISSUE TITLE
install service mode manually on Linux

### ISSUE THREAD

**Issue by Fndroid** [Fndroid/clash_for_windows_pkg#3464]
I have this issue on Linux too after  I access `sudo` permission, the app raises a permission error, if I know which packages need to activate service mode I can install it manually.

_Originally posted by @amir-valizadeh in https://github.com/Fndroid/clash_for_windows_pkg/issues/3461#issuecomment-1250717761_

**Comment by Fndroid** [comment]
@amir-valizadeh  steps to install Service Mode manually:
1. copy folder `APP HOME PATH/resources/static/files/linux/[x64|arm64]/service` to `Home Directory`(open from General) folder
2. create file `/usr/lib/systemd/system/clash-core-service.service` with content:
   ```
   [Unit]
   Description=Clash core service created by Clash for Windows
   After=network-online.target nftables.service iptabels.service
   
   [Service]
   Type=simple
   ExecStart=<path to Home Directory/service/clash-core-service>
   Restart=always
   RestartSec=5
   
   [Install]
   WantedBy=multi-user.target
   ```
   **you have to change the `ExecStart` path to `Home Directory/service/clash-core-service`!!!**
3. run commands:
   ```
   sudo systemctl enable clash-core-service
   ```
   ```
   sudo systemctl start clash-core-service
   ```

**Comment by Fndroid** [comment]
This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 7 days

**Comment by wjfwjfwjf** [comment]
does this is a spelling wrong?
[Unit]
Description=Clash core service created by Clash for Windows
After=network-online.target nftables.service iptabels.service



iptabels.service should be iptables.service?

**Comment by Fndroid** [comment]
@wjfwjfwjf 😄 I think you are right!

**Comment by otakutyrant** [comment]
I installed clash-for-windows-bin from AUR and I failed to install service mode in the app. It complains "User did not grant permission". I followed https://github.com/Fndroid/clash_for_windows_pkg/issues/3464#issuecomment-1250788591 to install service manually and activated the green earth icon successfully in the dashboard of cfw. It is simple to solve the issue in Arch Linux. No manuall copy. Just `sudoedit /usr/lib/systemd/system/clash-core-service.service` as below:

```
[Unit]
Description=Clash core service created by Clash for Windows
After=network-online.target nftables.service iptables.service

[Service]
Type=simple
ExecStart=/opt/clash-for-windows-bin/resources/static/files/linux/x64/service/clash-core-service
Restart=always
RestartSec=5

[Install]
WantedBy=multi-user.target
```

And then use systemd to enable and start the service.

I think maybe the developer of cfw is not good at making this process automatic and internal in Linux. So this is a workaround.
